test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1278,6 +1286,11 @@ class select_ep_transport : public test_ucp_wireup {
         modify_config("NUM_EPS", num_eps1);
         entity *e1 = create_entity();
 
+        if (!has_resource(*e1, "dc_mlx5") && !has_resource(*e1, "rc_mlx5") &&
+            !has_resource(*e1, "rc_verbs")) {
+            UCS_TEST_SKIP_R("RMA transports are not present");
+        }
+
```

Comment:
can we instead change the way the tests are instantiated? Like it is done for `test_ucp_wireup_unified`

Reply:
actually skipping like I do is too restrictive (posix can be supported), updated the condition.